sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -4045,6 +4045,38 @@ goog.scope(function() {
     * @param {number} internalFormat
     * @param {number} width
     * @param {number} height
+    */
+    sglrReferenceContext.ReferenceContext.prototype.texImage2DDelegate = function (target, level, internalFormat, width, height) {
```

Comment:
There is already a function named texImage2D in sglrReferenceContext.ReferenceContext.prototype.texImage2D. Due JS doesn't support overload, so this name is used here.